src/tests/test_rtc_data_models_aec.py

```diff
@@ -456,15 +457,14 @@ def test_validate_dte_tipo_dte(self) -> None:
         self._set_obj_1()
 
         obj = self.obj_1
-        expected_validation_errors = [
-            {
-                'loc': ('dte',),
-                'msg': """('Value is not "cedible".', <TipoDte.NOTA_CREDITO_ELECTRONICA: 61>)""",
-                'type': 'value_error',
-            },
-        ]
-
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        expected_validation_error = {
+            'loc': ('dte',),
+            'msg': 'Value error, (\'Value is not "cedible".\', '
+            '<TipoDte.NOTA_CREDITO_ELECTRONICA: 61>)',
```

Comment:
(⚠️ This review comment applies to all similar cases.)

Reformat and prettify string.

```suggestion
            'msg': (
                "Value error,"
                """ ('Value is not "cedible".', <TipoDte.NOTA_CREDITO_ELECTRONICA: 61>)"""
            ),
```